### PR TITLE
Mosaico/desktop view for static pages

### PR DESCRIPTION
**Pull request by MicciKarkia**

### CODE REVIEW

File: less/mosaico/static-pages/bruksvillkor.less [hunk 1]
```diff
@@ -1,56 +1,56 @@
...
-    font-size: 1rem;
+  .terms__sub-section-headline {
+    font-size: 1.08rem;
   }
 
-  p {
-    margin-top: 10px;
-    margin-bottom: 10px;
+  .terms__sub-section-sub-headline {
+    font-size: 1.075;
```
Comment (danielrlc) [inline less/mosaico/static-pages/bruksvillkor.less:41]: Could this maybe be in `rem`, like the font-sizes used previously?

File: less/mosaico/static-pages/fragor-och-svar.less [hunk 2]
```diff
@@ -22,7 +24,7 @@
     flex-direction: column;
```
Comment (danielrlc) [inline less/mosaico/static-pages/fragor-och-svar.less:24]: Declared twice.

File: less/mosaico/static-pages/bruksvillkor.less (hunk 1 above)
Comment (MicciKarkia) [reply less/mosaico/static-pages/bruksvillkor.less:41]: It is supposed to be, my mistake, I'll fix it!

File: less/mosaico/static-pages/kundservice.less [hunk 3]
```diff
@@ -5,30 +5,45 @@
...
+      grid-template-columns: 1fr 1fr;
+      padding: 0;
+      gap: 20px 30px;
+      margin-top: 50px;
+    }
   }
 
   .customer-service__menu-button {
-    margin: 10px 0;
+    /*margin: 10px 0;*/
```
Comment (danielrlc) [inline less/mosaico/static-pages/kundservice.less:33]: Maybe remove?

**Review by danielrlc — APPROVED** [review]

File: less/mosaico/static-pages/kundservice.less (hunk 3 above)
Comment (MicciKarkia) [reply less/mosaico/static-pages/kundservice.less:33]: Yes, in the next PR it will be removed or activated

**Pull request merged**